app/models/subscriber.rb

```diff
@@ -0,0 +1,7 @@
+class Subscriber < ApplicationRecord
+  validates :email, uniqueness: { case_sensitive: false }, presence: true, email: { mode: :strict, require_fqdn: true }
+  validates :subscribed, inclusion: { in: [true, false], message: "must be true or false" }
```

Comment:
the presence validation does not work for a Boolean attr as it considers `false` not present, so they must be validated by an inclusion validation